### PR TITLE
Bug/sc 16473/edit profile does not register existing input

### PR DESCRIPTION
**Pull request by Maliyaa**
BUG FIXED by creating an empty form group to allow the user info to be captured in the ngOnInit before validating for 'required'.

### CODE REVIEW

File: src/app/cube/user-profile/user-profile.component.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export class UserProfileComponent implements OnInit {
   ) {}
 
   async ngOnInit() {
+    console.log('yo');
```
Comment (JosueMolinaMorales) [inline src/app/cube/user-profile/user-profile.component.ts:27]: Console log yo

**Review by JosueMolinaMorales — COMMENTED** [review]
Looks good! Just the console log.

File: src/app/core/auth-validation.service.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { AbstractControl, AsyncValidatorFn, FormControl, FormGroup, Validators } from '@angular/forms';
```
Comment (mfranz1) [inline src/app/core/auth-validation.service.ts:2]:
```suggestion
import { FormControl, FormGroup, Validators } from '@angular/forms';
```
If these imports are not being used, lets delete them.

**Review by mfranz1 — CHANGES_REQUESTED** [review]
Looks good! Just one change

**Review by pzalep1 — COMMENTED** [review]
After the console is removed and the imports are fixed this will be good to go!

**Review by mfranz1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Review by JosueMolinaMorales — APPROVED** [review]

**Review by pmanal1 — APPROVED** [review]

**Pull request merged**